.gitignore

```diff
@@ -6,7 +6,9 @@
 !/cmake/*.cmake
 *~
 /test/benchmark_test
+/test/benchmark_test.exe
```

Comment:
I think we shouldn't build in tree. I tried to follow the current `.gitignore` style that was specifically adding the built executables. I think a catchall `*.exe` would be much more useful.